control/grpc.py

```diff
@@ -1781,16 +1860,23 @@ def create_listener_safe(self, request, context):
 
             if enable_ha:
                   for x in range (MAX_ANA_GROUPS):
+                       _ana_state = "inaccessible"
+                       if self.ana_map[request.nqn]:
+                           _ana_state = "optimized" if  self.ana_map[request.nqn][x+1] == pb2.ana_state.OPTIMIZED else "inaccessible"
+                           self.logger.info(f"using ana_map: set listener on nqn : {request.nqn}  ana state : {_ana_state} for group : {x+1}")
                        try:
+                          self.logger.info(f"create_listener nvmf_subsystem_listener_set_ana_state {request=} {_ana_state=} anagrpid={x+1}")
```

Comment:
@baum I see we call the SPDK set ANA state function both here and in the set_ana_state() function. Is there a reason for this duplication?